triggers.lesson14/styles-for-triggers.css

```diff
@@ -0,0 +1,165 @@
+.container div {
+    display: flex;
+    margin: 150px;
+    padding: 10px 150px 10px 10px;
+    border: rgb(7, 69, 151) solid 2px;
+}
+
+.content div {
+    font-size: 20px;
+    padding: 0px;
+    border: 2px solid rgb(201, 63, 139);
+    margin: 10px;
+}
+
+.container {
+    position: absolute;
+    animation: slide-from-left 1s;
+    animation-duration: 3s;
+    animation-timing-function: ease-out;
+}
+
+@keyframes slide-from-left {
+    from {
+        left: -1000px;
+    }
+
+    to {
+        left: 0;
+    }
+}
+
+.tooltip {
+    position: relative;
+    display: inline-block;
+    cursor: default;
+}
+
+.tooltip .tooltip-top {
+    width: 150px;
+    visibility: hidden;
+    position: absolute;
+    border-radius: 5px;
+    background-color: silver;
+    padding: 5px;
+    text-align: center;
+    bottom: 160%;
+    left: -40%;
+    opacity: 0;
+    transition: opacity 0.7s, visibility 0.7s;
```

Comment:
I suppose the visibility and opacity in your case play the same role so you could use just one of them.

Reply:
I use opacity for the smooth effect, but I need hidden so that the tooltip does not appear when the cursor is not on the trigger.
Now I'm trying to get a smooth effect from hidden only and I can't...